libs/@blockprotocol/graph/src/graph-module.json

```diff
@@ -696,20 +204,20 @@
       "data": {
```

Comment:
We're missing a variant here:
```ts
     "data": {
        "oneOf": [
          {
            "type": "object",
            "properties": {
              "description": {
                "type": "string"
              },
              "name": {
                "type": "string"
              },
              "url": {
                "type": "string",
                "format": "uri",
                "$comment": "The URL to take the file from, if not being uploaded directly. 
              }
            },
            "required": [
              "url"
            ]
          },
          {
            "type": "object",
            "properties": {
              "description": {
                "type": "string"
              },
              "name": {
                "type": "string"
              },
              "file": {
                "$comment": "the file blob, if being uploaded directly."
              }
            },
            "required": [
              "file"
            ]
          }
        ]
      },
```

Reply:
[Add missing variant of uploadFile](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/e601b5cad362ba2559d51f8b5ff144b9a186ae17)